jetty-io/src/main/java/org/eclipse/jetty/io/ChannelEndPoint.java

```diff
@@ -94,14 +88,7 @@ public void close()
         }
     }
 
-    private final ManagedSelector.SelectorUpdate _updateKeyAction = new ManagedSelector.SelectorUpdate()
-    {
-        @Override
-        public void update(Selector selector)
-        {
-            updateKey();
-        }
-    };
+    private final ManagedSelector.SelectorUpdate _updateKeyAction = this::updateKey;
```

Comment:
This is not used in `updateKey(Selector selector)`  or is it the other update key?   Eitherway, it is confusing, can we change one of the method names?

Reply:
I don't understand what's confusing? Like before the inner class was calling `Selectable.updateKey()`, now the method reference is eventually calling `Selectable.updateKey()` - so it's the same stuff and it's the same task of updating the key.

Reply:
there are two methods both called `updateKey`, both are used as method lambdas where you can't see the args without looking elsewhere.  One is set here as a field, the other is passed much later on.    Sorry, it is just confusing and even when I've worked it out, I keep getting reconfused.   Rename one of the methods.